config/routes.rb

```diff
@@ -39,7 +39,7 @@
   get '/forum', to: redirect('https://forum.ruby.org.au'), as: :forum
   get '/mailing-list', to: redirect('https://confirmsubscription.com/h/j/3DDD74A0ACC3DB22'), as: :roro_mailing_list
   get '/slack', to: redirect('https://join.slack.com/t/rubyau/shared_invite/zt-1pewt4vi8-TtrM~UoIJmuH9Niy0Ela6w'), as: :slack
-  get '/videos', to: redirect('https://www.youtube.com/channel/UCr38SHAvOKMDyX3-8lhvJHA/videos'), as: :videos
+  get '/videos', to: redirect('https://www.youtube.com/@RubyAustralia'), as: :videos
```

Comment:
Vanity URL needs to include /videos:
> https://www.youtube.com/@RubyAustralia/videos

Otherwise people get dumped at the “profile” page which doesn’t show any videos…

Reply:
My test of https://www.youtube.com/@RubyAustralia/ takes me to a page that lists the top videos and then latest videos, which I feel is better than just the latest. 

Can you screenshot what you're describing because it sounds different to what I'm seeing.

Reply:
OK, it's just me. I tried the base URL both logged into YT and not logged in - same result. Then tried FF and Edge - works as you've described. Go back to my browser; couple of web searches and "check your extensions" was next on the list. Create browser profile with no web extensions, and YT works fine. So while I haven't figured out what's responsible (it's probably my ad blocker but can't confirm) - I'll put my hand up and say I should have checked this more thoroughly before flagging it as an issue.

IOW - 🚢 ship it.